src/components/utils/utils.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const ncNews = axios.create({
+  baseURL: "https://nc-project-pmrs.onrender.com/api",
+});
+
+export const fetchArticles = () => {
+  return ncNews.get("/articles").then(({ data }) => {
+    return data;
+  });
```

Comment:
Nice, but instead of just sending data like this, we should instead be sending data.articles here, so the array of articles is directly what we have access to on the receiving end.